packaging/images/openshift-ci/Dockerfile

```diff
@@ -8,7 +8,14 @@ WORKDIR $GOPATH/src/github.com/redhat-et/microshift
 
 COPY . .
 
-RUN yum install gpgme-devel glibc-static libassuan-devel -y
+RUN yum install gpgme-devel glibc-static libassuan-devel wget -y
+
+# update to go-1.18
```

Comment:
Is go-1.18 the right version for 4.10? Looks like 4.11 is using 1.17 still.

Reply:
great, yea, probably so, will do

Reply:
updated!